.travis.yml

```diff
@@ -93,6 +94,9 @@ before_install:
       sed -e '/^coala/d' requirements.txt > requirements.new
       mv requirements.new requirements.txt
     fi
+  # Install astyle linter
+  - sudo apt-get -qq update
+  - sudo apt-get install -y astyle
```

Comment:
It was suggested that my builds are failing on travis because it does not have astyle installed